ui/jobs/jobs.js

```diff
@@ -1108,6 +1108,17 @@ class Bars {
     ];
   }
 
+  uiDisplay(options) {
+    const bars = document.getElementById('bars');
+    if (bars) {
+      const barList = bars.children;
+      for (const bar of barList) {
+        if (bar.id == 'hp-bar' || bar.id == 'mp-bar') continue;
```

Comment:
`===` unless there's a reason not to.  I know code isn't consistent elsewhere and it's not safe to globally change them all, but try to use `===` in new code.